src/utils/colours.ts

```diff
@@ -1,6 +1,10 @@
 const darkGrey = "#62696D"
+const gdsLightGrey = "#f3f2f1"
+
 const tagBlue = "#e9f1f8"
 const textBlue = "#114e81"
-const gdsLightGrey = "#f3f2f1"
+const navy = "#1D70B8"
```

Comment:
I think it's nice that we extracted the blue colour, but maybe we could follow the gov UK naming, which is blue? https://design-system.service.gov.uk/styles/colour/

Reply:
Thanks for sharing the link. I'll rename `navy` to `blue`.